src/test.js

```diff
@@ -258,6 +258,7 @@ Test.prototype = {
 		}
 
 		emit( "testEnd", this.testReport.end( true ) );
+		this.testReport.slimAssertions();
```

Comment:
They're unrelated. The assertion objects used by the new js-reporters events are not the same as the ones used for the old callbacks.